modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/ListViewBehavior.java

```diff
@@ -563,8 +577,11 @@ private void selectPreviousRow() {
             return;
         }
 
+        MultipleSelectionModel<T> selectionModel = getNode().getSelectionModel();
+        if (selectionModel == null) return;
```

Comment:
VERY MINOR style issue: this seems to be the only place where you use a _one-liner_ return

Reply:
Yes, this is unfortunately different everywhere in JavaFX and I usually just adapt to the already existing code, but in this case I didn't. Just changed it.